HealthAPILibrary/GiniHealthAPILibrary/Sources/GiniHealthAPILibrary/Documents/ExtractionResult.swift

```diff
@@ -20,7 +25,11 @@ import Foundation
     
     /// The line item compound extractions.
     public var lineItems: [[Extraction]]?
-        
+
+    public var isPayable: Bool {
+        extractions.first(where: {$0.name == "payment_state"})?.value == PaymentState.payable.rawValue
```

Comment:
Proposition: Maybe we can define enum with extraction names?

Reply:
Thanks for the proposition. Adopted!